docs/admin/configure.md

```diff
@@ -55,7 +74,7 @@ configure the server by setting the following variables in `/etc/coder.d/coder.e
 CODER_ACCESS_URL=https://coder.example.com
 
 # String. Address to serve the API and dashboard.
-CODER_ADDRESS=127.0.0.1:3000
+CODER_HTTP_ADDRESS=127.0.0.1:3000
```

Comment:
```suggestion
CODER_HTTP_ADDRESS=0.0.0.0:3000
```

Reply:
I'm pretty sure the default is the loopback address, according to the help text.